internal/errors/errdot_119.go

```diff
@@ -0,0 +1,20 @@
+//go:build go1.19
```

Comment:
Wouldn't `internal/backend` be a more suitable place? We only every check that error after using `internal/backend/foreground_*.go`/

Reply:
I started out in `internal/backend` but when I found that there was a package that explicitly was about errors, I figured that a function checking an error for a specific type of error belonged in there instead :-) It's entierly up to you where to keep it (until Go 1.19 is the minimum supported Go version in restic), do you still want me to change it?

Reply:
IMO the error package should only be used for generic error handling code. But the `ErrDot` error is really only relevant for the backend code. So from a code cohesion perspective, the this file belongs into `internal/backend`. So yes, please change the location.